LoginServer/SaltGenerator.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
```

Comment:
fun thing:
another piece of code that could be shared.  When you salt and encrypt the user details Before sending them to the server.
Just need to fix this so it generates the same salt, given a set of known shared details.